types/EdgeDataApi.ts

```diff
@@ -3,4 +3,6 @@ export type EdgeDataApi = {
   projectId: string;
   source: string; // Source node ID
   target: string; // Target node ID
+  edgeValue?: string;
+  type?: string;
```

Comment:
We are not storing edge type in the database as far as i know

Reply:
remove